keyboards/handwired/dactyl_manuform/dmote/62key/keymaps/default/keymap.c

```diff
@@ -0,0 +1,152 @@
+                        _______, _______, _______, _______, _______,
+      _______, _______,          _______,
+  _______, _______, _______,
+      _______, _______
+),
+
+[_NUMERIC] = LAYOUT_62KEY(
+  LAYER_C, KC_INS,  KC_F2,   KC_F3,   KC_F4,   KC_F5,
+  KC_F12,  KC_F1,   KC_2,    KC_3,    KC_4,    KC_5,
+  _______, KC_1,    KC_AT,   KC_HASH, KC_DLR,  KC_PERC,
+  KC_GRV,  KC_EXLM, KC_BTN1, KC_WH_U, KC_BTN2,
+                             KC_WH_D,          RGB_MOD, _______,
+                                           _______, _______, EMDASH,
+                                               _______, _______,
+
+               KC_F6,   KC_F7,   KC_F8,   KC_F9,   RESET,   KC_WAKE, // *
+               KC_6,    KC_7,    KC_8,    KC_9,    KC_F10,  KC_F11,
+               KC_CIRC, KC_AMPR, KC_ASTR, KC_APP,  KC_0,    PASTE,
+                        KC_MS_L, KC_MS_U, KC_MS_R, KC_PSCR, RGB_TOG,
+      KC_ACL1, KC_ACL2,          KC_MS_D,
+  KC_ACL0, _______, _______,
+      _______, _______
+)
+};
+// *KC_WAKE: Used in place of KC_SLEP because X11 with i3 on prerelease
+//  Debian 10 was seeing duplicate keypress and release events for sleep
+//  (regardless of i3 binding), which ruined the function.
+
+
+/*
+The rest is all about lighting control.
+The logic here represents a pretty poor compromise solution between the
+following concerns:
+
+- Feedback on active modifiers.
+- Flexibility: Both sides of the keyboard are interchangeable.
+- Regular QMK RBG lighting modes. Specifically, Knight and Xmas.
+
+Currently, the last item suffers, because the first two seem to require calling
+a function that implements the RGBLIGHT_SPLIT_SET_CHANGE_HSVS macro, which
+most of the rgblight.c functions do not. In particular, functions that target
+an individual LED do not do so correctly across the wire, so instead we
+let HSV vary without ever targeting LEDs.
+*/
+
+// How long to wait between animation steps for "Knight" animation:
+const uint8_t RGBLED_KNIGHT_INTERVALS[] PROGMEM = {255, 200, 100};
+
+bool _initialized = false;
+bool _leds_dirty = false;
+
+void update_leds(void) {
+  uint8_t mods = get_mods();
+  uint16_t hue = 355;  // Rough match to printed case.
+  uint8_t saturation = 255;
+  uint8_t value = 0;
+  if (layer_state_is(_COLEMAK)) { hue -= 50; saturation -= 20; value += 20; };
+  if (layer_state_is(_NUMERIC)) { value += 30; };
+  if (mods & MODS_SHIFT_MASK) { saturation -= 20; value += 30; };
+  if (mods & MODS_ALT_MASK) { hue -= 100; saturation -= 20; value += 30; };
+  if (mods & MODS_CTRL_MASK) { hue -= 200; saturation -= 20; value += 30; };
+  // rgblight_sethsv_eeprom_helper is not a great API function but it does
+  // affect both halves of a split keyboard.
+  rgblight_sethsv_eeprom_helper(hue, saturation, value, false);
+  _leds_dirty = false;
+}
+
+void matrix_init_user(void) {
+}
+
+void matrix_scan_user(void) {
+  if (_leds_dirty) { update_leds(); };
+}
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
## Lines 135-138:

You don't seem to be using this function at all, and it's been deprecated in favor of [`set_single_persistent_default_layer(layer)`](https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer).